packages/babel-plugin/src/styled/index.tsx

```diff
@@ -1,9 +1,11 @@
 import * as t from '@babel/types';
 import type { NodePath } from '@babel/core';
+
+import type { Metadata, Tag } from '../types';
```

Comment:
Can we turn a lint rule on for imports?

Reply:
For sorting and grouping? I was planning to, otherwise not sure what you're referring to

Reply:
Yeah exactly